plex-argo/README.md

```diff
@@ -54,6 +70,7 @@ ArgoURL=$(curl -s http://localhost:33400/metrics | grep -oP 'userHostname="https
 
 if [[ $ArgoURL != $PlexAPIcustomConnections ]]; then
     docker container stop plex
+    #systemctl stop plexmediaserver.service
```

Comment:
Lets just make two bash files.
Plex-Argo-DirectoryUpdate-systemd.bash and Plex-Argo-DirectoryUpdate-docker.bash